### DISCUSSION TITLE
Ironing temperature

### DISCUSSION THREAD

**Discussion by mmiguel4** [supermerill/SuperSlicer#2330] [Q&A] [answered]
Is there anyway to use a different temperature for ironing? In large parts I have have clogs after about 30min or so ironing with PLA. Tried to change the temperature the temperature from 205ºC to 180ºC when ironing start manually and it never happened again.  It was nice to have the option to do that in slicer.

**Comment by supermerill** [comment, marked as answer]
No temp for feature, as the time to change the temp is way too long. You'll be already to the next feature when the heating/cooling is finished.

If you still want to play with it, you can use the `feature_gcode` setting with something like:
`{if extrusion_role == "ironing"}M104 S{temperature-20}{else}M104 S{temperature}{endif}`

btw, the problem with ironing is often when there is too many retraction, as it's almost not extruding (also heat creep). Try changing the setting to extrude more in the ironing phase, or/and disable retraction for non-perimeter crossing.

**Comment by mmiguel4** [comment]
The part with the problem is box shaped with large flat top, it took almost an how to iron it. Retractions were already disabled inside the perimeters. 
I will play a bit and see...
The place for that code is in "Between extrusion role change G-code, right?

**Reply by supermerill** [reply]
yes

**Comment by mmiguel4** [comment]
Thanks
I will try this week in practice and report if my problem is solved.

**Comment by mmiguel4** [comment]
Didn't workout as expected, after all the problem was not a heat creep  clog. The top of the part was slightly dropping down after cooling a bit during ironing phase. Since the nozzle was farther from the part it looked like under extrusion. 

Anyway, the custom feature GCode worked as expected.
